Makefile

```diff
@@ -1,5 +1,5 @@
 .DEFAULT_GOAL = help
-.PHONY: help install gitmodules build start stop wait-healthy sh exec logs watch test run dev prod
+.PHONY: help install gitmodules build start stop wait-healthy sh exec logs watch test test\:mutation run dev prod
```

Comment:
Waiting on discovery in https://github.com/libero/article-page/pull/4#discussion_r400260656.